build-tools/unix-distribution-setup/unix-distribution-setup.mdproj

```diff
@@ -0,0 +1,26 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ItemType>GenericProject</ItemType>
+    <ProjectGuid>{8FF78EB6-6FC8-46A7-8A15-EBBA9045C5FA}</ProjectGuid>
```

Comment:
Projects need to have unique GUIDs within the solution, and this is the same guid that `android-toolchain.mdproj` uses.